cmd/catalog-importer/cmd/sync.go

```diff
@@ -161,8 +161,8 @@ func (opt *SyncOptions) Run(ctx context.Context, logger kitlog.Logger, cfg *conf
 				"catalog_type_id", existingCatalogType.Id,
 			)
 
-			for _, output := range cfg.Outputs() {
-				if output.TypeName == existingCatalogType.TypeName {
+			for _, confType := range cfg.AllOutputTypes() {
+				if confType.TypeName == existingCatalogType.TypeName {
```

Comment:
This is the key fix! Previously we were only checking top-level types, but not looking at any enum types inferred from them.